config.go

```diff
@@ -399,6 +399,13 @@ func loadConfig() (*config, []string, error) {
 		return nil, nil, err
 	}
 
+	// Ensure a valid payment method is set.
+	if cfg.PaymentMethod != dividend.PPS && cfg.PaymentMethod != dividend.PPLNS {
```

Comment:
This doesn't apply if the pool is solo pool mining. Also the payment method defaults to PPS if not set in the config, this check is not needed.